stubs/docutils/docutils/nodes.pyi

```diff
@@ -1,15 +1,75 @@
+import xml.dom.minidom
 from _typeshed import Self
-from collections.abc import Iterable
-from typing import Any, overload
+from collections.abc import Iterable, Sequence
+from typing import Any, Callable, Generator, Protocol, TypeVar, overload
+from typing_extensions import Literal
 
 from docutils.transforms import Transformer
 
+class NodeVisitor:
+    def __init__(self, document: document): ...
+    def __getattr__(self, __name: str) -> Any: ...  # incomplete
+
+_N = TypeVar("_N", bound=Node)
+
+class _DomModule(Protocol):
+    Document: type[xml.dom.minidom.Document]
+
 class Node:
+    children: Sequence[Node]
```

Comment:
We should add a note that this is never initialized on `Node` itself, but by its subclasses.